src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstance.cs

```diff
@@ -304,12 +304,6 @@ private void StartRemoteIntegrationService(DTE dte)
             {
                 dte.ExecuteCommand(WellKnownCommandNames.Test_IntegrationTestService_Start);
             }
-
-            if (AsyncCompletionCondition.Instance.ShouldSkip
-                && dte.Commands.Item(WellKnownCommandNames.Test_IntegrationTestService_DisableAsyncCompletion).IsAvailable)
```

Comment:
❕  This command needs to be removed from WellKnownCommandNames and from the VSCT.

Reply:
📝 This is still problematic. Here are the locations I know need to be updated. There may be additional references that become obvious when you try to remove them.

https://github.com/dotnet/roslyn/blob/1689a20df6d51e123e7fc028b4a4cad6d8370eba/src/VisualStudio/IntegrationTest/TestSetup/IntegrationTestServiceCommands.vsct#L71
https://github.com/dotnet/roslyn/blob/1689a20df6d51e123e7fc028b4a4cad6d8370eba/src/VisualStudio/IntegrationTest/TestSetup/IntegrationTestServiceCommands.vsct#L46-L54
https://github.com/dotnet/roslyn/blob/1689a20df6d51e123e7fc028b4a4cad6d8370eba/src/VisualStudio/IntegrationTest/TestSetup/IntegrationTestServiceCommands.cs#L29
https://github.com/dotnet/roslyn/blob/1689a20df6d51e123e7fc028b4a4cad6d8370eba/src/VisualStudio/IntegrationTest/TestSetup/IntegrationTestServiceCommands.cs#L72-L78
https://github.com/dotnet/roslyn/blob/1689a20df6d51e123e7fc028b4a4cad6d8370eba/src/VisualStudio/IntegrationTest/TestSetup/IntegrationTestServiceCommands.cs#L43
https://github.com/dotnet/roslyn/blob/1689a20df6d51e123e7fc028b4a4cad6d8370eba/src/VisualStudio/IntegrationTest/TestSetup/IntegrationTestServiceCommands.cs#L157-L169